### ISSUE TITLE
[REQ] Ability to sort by provided property 

### ISSUE THREAD

**Issue by stramel** [Hackception/hc-tree-view#40]
Current implementation sorts by `parentId`, `nodeId`. It has been requested that the tree be able to be organized by other properties such as names A-Z

**Comment by admwx7** [comment]
It's a tree, it will always be a parent -> child relationship, is this a request to add sorting of nodes?

**Comment by stramel** [comment]
Yes, so tree structure obviously by `parentId`, `nodeId` but then within a branch the leaves are sorted by the `nodeId`. The request is to be able to sort them by other properties such as the `label` A-Z

**Comment by admwx7** [comment]
Right now it's actually not explicitly sorted in anyway, whatever order it's provided in is the order it's displayed in.  We can add sorting support, but it's a bit tricky since we'll have to do it in the JS instead of just using the sort functionality built into iron-list, since once it hits the iron-list it's flattened and we need the parent node to still be above the children nodes.
